docs/developer/editor.rst

```diff
@@ -0,0 +1,85 @@
+##############
+ORD Web Editor
+##############
+
+The ORD hosts an interactive web editor for ``Reaction`` and ``Dataset`` protos
+on Google Cloud Platform (GCP); it is available at
+`https://editor.open-reaction-database.org`.
+
+This document describes the setup and deployment process for GCP.
+
+***********************
+Update the docker image
+***********************
+
+If the editor code has changed, you will need to update the docker image on
+Docker Hub (`link
+<https://hub.docker.com/repository/docker/openreactiondatabase/ord-editor>`_).
+Before starting, make sure that you have installed all of the required
+`dependencies
+<https://github.com/Open-Reaction-Database/ord-schema/blob/main/editor/README.md#dependencies>`_.
+
+.. NOTE::
+   This updates the ``latest`` tag for the ord-editor image. To set a different
+   tag add ":<tag>" to the end of the name; for example
+   ``openreactiondatabase/ord-editor:v0.0.0``.
+
+.. code-block:: shell
+
+    $ cd "${ORD_SCHEMA_ROOT}/editor"
+    $ make package
+    $ docker build -t openreactiondatabase/ord-editor .
+    $ docker push openreactiondatabase/ord-editor
+
+************************
+Create a new VM instance
+************************
+
+.. IMPORTANT::
+   If you are only updating the docker image, **you only need to reset the
+   existing VM rather than create a new one!**
+
+.. NOTE::
+   Only one instance at a time can access a persistent disk. Make sure to shut
+   down (and delete) any existing instance before proceeding.
```

Comment:
In addition to shut-down and delete, is any cleanup necessary for the instance group?

Reply:
Yes, updated.